core/lib/navigation-error.js

```diff
@@ -16,6 +16,13 @@ const UIStrings = {
    */
   warningXhtml:
     'The page MIME type is XHTML: Lighthouse does not explicitly support this document type',
+  /**
+   * Warning shown in report when the page under test returns an error code, which Lighthouse is not able to reliably load
+   * so we display a warning.
+   */
+  warningStatusCode: 'Lighthouse was unable to reliably load the page you requested. Make sure' +
+    ' you are testing the correct URL and that the server is properly responding' +
+    ' to all requests.',
```

Comment:
I think having the status code in the string was useful. You can add dynamic values to UI strings like [this](https://github.com/GoogleChrome/lighthouse/blob/main/core/gather/driver/navigation.js#L170).